microservices-connector/internal/controller/gmconnector_controller.go

```diff
@@ -128,9 +129,16 @@ func createService(step string, ns string, svc string, svcCfg *map[string]string
 
 		gvr, _ := meta.UnsafeGuessKindToResource(*gvk)
 		for {
-			// _, err = dynamicClient.Resource(gvr).Namespace(ns).List(context.TODO(), metav1.ListOptions{})
+			patchBytes, err := json.Marshal(obj)
+			if err != nil {
+				return fmt.Errorf("failed to marshal config to json: %v", err)
+			}
 
-			createdObj, err := dynamicClient.Resource(gvr).Namespace(ns).Create(context.TODO(), obj, metav1.CreateOptions{})
+			// createdObj, err := dynamicClient.Resource(gvr).Namespace(ns).Create(context.TODO(), obj, metav1.CreateOptions{})
```

Comment:
Does this commented code help document something or something we may want to revert back to?

Reply:
I think this line can be deleted